### ISSUE TITLE
Support 1D signals

### ISSUE THREAD

**Issue by andrefaraujo** [google-research/receptive_field#3]
See initial discussion in https://github.com/google-research/receptive_field/issues/1

@Permafacture offered to help with this.

The main thing to be done would be to generalize/abstract the layer parsing functions to handle 1D (besides 2D): get_layer_params, _stride_size, _conv_kernel_size, etc in https://github.com/google-research/receptive_field/blob/master/receptive_field/python/util/parse_layer_parameters.py#L41

From the discussion in https://github.com/google-research/receptive_field/issues/1, we see that the 1D conv may be done as a Conv2D with ExpandDims (from @Permafacture:
`ValueError: Weight layer's name input to conv layer does not end with '/read' or '/Conv2D/ReadVariableOp': projection_layer/conv1d/ExpandDims_1`)

The most efficient way to make progress here is to create a new test network for a 1D case in https://github.com/google-research/receptive_field/blob/master/receptive_field/python/util/parse_layer_parameters_test.py, this will help understand if a proposed implementation is parsing the parameters correctly.

**Comment by Permafacture** [comment]
Should I make a pull request that adds that test network?

**Comment by andrefaraujo** [comment]
Hmm, I guess a test using the 1D network wouldn't pass. So I'd suggest to only send a pull request once the code is adapted to also parse correctly parameters for the 1D case.

**Comment by okovalchuk98** [comment]
Hi. @andrefaraujo, do you have any updates about support 1D signal?

**Comment by andrefaraujo** [comment]
Not really, @okovalchuk98 . Would you be interested in helping with it?
